### PR TITLE
Have even read-only servers leave session loaded.

### PR DESCRIPTION
**Pull request by mtbc**
Fix https://github.com/ome/omero-server/issues/76 by having even read-only servers maintain a loaded session. Smoketest on an actual [read-only server](https://docs.openmicroscopy.org/omero/5.6/developers/Server/Clustering.html#read-only) before approving.

### CODE REVIEW

**Comment by mtbc** [comment]
Will have to check `SecuritySystemTest`, oddly it doesn't run locally for me with `gradle test`.

**Comment by mtbc** [comment]
This breaks `TestIShare.test_basic_usage` from `test_ishare.py`.

File: src/main/java/ome/security/basic/BasicSecuritySystem.java [hunk 1]
```diff
@@ -553,7 +558,8 @@ public void loadEventContext(boolean isReadOnly, boolean isClose) {
         }
 
         final ome.model.meta.Session session;
-        if (isReadOnly) {
+        if (isShare) {
```
Comment (joshmoore) [inline src/main/java/ome/security/basic/BasicSecuritySystem.java:561]: I'm unsure where `isReadOnly` has gone.

Comment (mtbc) [reply src/main/java/ome/security/basic/BasicSecuritySystem.java:561]: Following from https://github.com/openmicroscopy/omero_server_private/pull/5/files#r363291960 I found that it probably is unnecessary for read-only.

Comment (joshmoore) [reply src/main/java/ome/security/basic/BasicSecuritySystem.java:561]: After a chat, I now understand that this refers to my previous comment that `isReadOnly` was likely an unneeded performance optimization and can be removed.

**Pull request merged**